src/sentry/deletions/base.py

```diff
@@ -134,41 +153,50 @@ def delete_bulk(self, instance_list) -> bool:
 
         for instance in instance_list:
             child_relations = self.get_child_relations(instance)
-            child_relations = self.extend_relations(child_relations, instance)
             child_relations = self.filter_relations(child_relations)
             if child_relations:
                 has_more = self.delete_children(child_relations)
                 if has_more:
                     return has_more
 
-        return self.delete_instance_bulk(instance_list)
+        self.delete_instance_bulk(instance_list)
 
-    def delete_instance(self, instance):
+        return False
```

Comment:
What does `False` mean for this function? That it did not delete?

Reply:
The `delete_bulk` method returns a boolean to indicate if there are more records to delete. This return value is mostly unused as `chunk()` uses a separate query to determine if all the data is removed.

https://github.com/getsentry/sentry/blob/b2e0b6e5ba549e28ccbeca3b78f28f7db216d701/src/sentry/deletions/base.py#L193-L214